src/mixable.ts

```diff
@@ -10,7 +10,12 @@ export default class Mixable {
 
 export interface MixResult {
     results: Result[]
+    type: MixableType,
```

Comment:
`MixableType` is defined for input Mixeable types, not `MixResult` type. When you mix Premix with Nicoshot what will be the type ? When you mix Premix with Premix what will be type ? When you mix Liquid with Nicoshot what will be the type ? When you mix Premix with Premix what will be the type ? etc...